.travis.yml

```diff
@@ -15,7 +15,7 @@ matrix:
       os: linux
 
 before_install:
-  - echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
+  - echo '{"ipv6":true, "fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
```

Comment:
And after `:` as in `"ipv6": true` (instead of `"ipv6":true`) ?

pleeeaaase? :smile: 
Then it should be :ok_hand: